demos/layout-grid.html

```diff
@@ -0,0 +1,178 @@
+<!DOCTYPE html>
```

Comment:
The autoprefixer bug has already been fixed and a new version released (!), so I've removed the workaround and bumped up the required autoprefixer version in `package.json`.